client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/services/BookmarkDB.java

```diff
@@ -38,6 +38,7 @@ public class BookmarkDB extends SQLiteOpenHelper
 	static final String DB_KEY_SCREEN_RESOLUTION = "resolution";
 	static final String DB_KEY_SCREEN_WIDTH = "width";
 	static final String DB_KEY_SCREEN_HEIGHT = "height";
+	static final String DB_KEY_SCREEN_SCALE = "scale";
```

Comment:
you need to increment the database version and add an update path for the new field.